### ISSUE TITLE
Package failed to updated on web after release.

### ISSUE THREAD

**Issue by quabug** [openupm/openupm#1327]
I've release [several packages](https://github.com/quabug/EntitiesBT/releases) at the same time by [action](https://github.com/quabug/EntitiesBT/actions/runs/260753485).
But [one](https://github.com/quabug/EntitiesBT/releases/tag/builder.component%2Fv0.21.3) of them failed to update itself on [web](https://openupm.com/packages/com.quabug.entities-bt.builder.component/).
And the [gitTagPrefix](https://github.com/openupm/openupm/blob/07c99538127c9ee917940efb801d3371d366766d/data/packages/com.quabug.entities-bt.builder.component.yml#L13) seems fine and haven't changed from last release.

**Comment by favoyang** [comment]
Thanks for reporting the issue. I'll get back to you, but a bit later today.

**Comment by favoyang** [comment]
TLDR; the problem has been fixed. It was my mistake during a data fix.

Details here...

The problem is that the pipelines received tag v0.21.3 (https://github.com/quabug/EntitiesBT/commit/b7d9f49a5f2cfab7a291f2fb4aa801dd7aa7e0c6) first and built for it. Then the PR for updating gitTagPrefix of your packages got merged. I noticed the issue and tried to fix it without bothering you, by removing wrong builds, pipeline records, and then clean the CDN. Well, turns out here a mistake you caught that I forgot to clean the build-pipelines record for builder.component-0.21.3. When the `builder.component/v0.21.3` tag was detected it was ignored because build-pipelines think that the version was built already. It has been fixed right now: https://openupm.com/packages/com.quabug.entities-bt.builder.component/. Sorry for the inconvenience.
